devfiles/java-web-vertx/meta.yaml

```diff
@@ -2,5 +2,5 @@
 displayName: Java Vert.x
 description: Java stack with OpenJDK 8 and Vert.x demo application
 tags: ["Java", "OpenJDK", "Maven", "Vertx"]
-icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+icon: ./images/vertx.svg
```

Comment:
Since current directory `./` is used here, I expect that it should be resolved to `CHE_PLUGIN_REGISTRY_HOST`/devfiles/java-web-vertx/images/vertx.svg...
But in fact, it will be ``CHE_PLUGIN_REGISTRY_HOST`/images/vertx.svg...`, right?
If yes - can we use reference relative to root folder, like `/images/vertx.svg` ?

Reply:
I actually had `/images/vertx.svg` initially, but it looks like the dashboard is just prepending the devfile registry url -- viewing the requests in the console showed images being loaded from e.g.
```
http://che-devfile-registry.cluster.net//images/go.svg
```
That may be a dashboard issue though -- I'm not sure.